RELEASE_NOTES.rst

```diff
@@ -24,6 +24,43 @@
 
 .. towncrier release notes start
 
+Airflow 3.0.6 (2025-08-25)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+No significant changes.
+
+Bug Fixes
+"""""""""
+
+- Fix Connection extra field masking errors when connections use masked values (#54780)
+- UI: Fix ``TriggerDagRunLink`` broken page when clicking "Triggered DAG" button (#54760)
+- Fix remote logging connection availability in Task SDK supervisor when connections are created via UI (#54720)
+- Fix ``task_queued_timeout`` not working after first DAG run by properly resetting ``queued_by_job_id`` (#54604)
+- Fix DAG version determination to use bundle path and relative fileloc instead of absolute fileloc (#54483)
+- Remove Kerberos replay cache (``KRB5CCNAME`` env) when running tasks with user impersonation (#54672)
+- Skip additional span-related database queries when tracing is disabled (#54626)
+- Fix ``max_active_tasks`` persisting after removal from DAG code (#54639)
+- UI: Automatically switch to the triggered DAG run in Graph/Grid view when manually triggering a DAG run (#54336)
+- UI: Fix "Maximum update depth exceeded" errors in Task Log Preview by filtering out empty log entries (#54628)
+- Fix custom logging configuration failures preventing triggerer and scheduler startup with simple module paths (#54686)
+
+Miscellaneous
+"""""""""""""
+
+- Add logging when triggerer reaches maximum trigger capacity for better observability (#54549)
+- Bump ``axios`` UI dependency from ``1.8.0`` to ``1.11.0`` (#54733)
+- Bump ``pluggy`` to ``1.6.0`` (#54728, #54730)
+
+Doc Only Changes
+""""""""""""""""
+
+- Fix broken link for Listener spec (#54535)
+- Remove experimental status from ``get_parsing_context`` function (#54802)
```

Comment:
Not important, but this feels odd to me to include in a patch release 🤷

Reply:
Yeah that -- should have been marked non-experimental since 3.0.0